server/lib/donations.js

```diff
@@ -32,7 +31,7 @@ export const processDonation = (Sequelize, donation) => {
                   groupId: group.id,
                   groupName: group.name,
                   paymentMethodId: paymentMethod.id,
-                  description: `OpenCollective: ${group.slug}`
+                  description: `https://opencollective.com/${group.slug}`
```

Comment:
Do you have the interval passed to that function? If so it would be great to add it to the description like we did in our db ("Monthly donation to ..."). I'd also consider appending the description of the donation since we let people donate to a custom tier (with the request donation URL).